src/app/icd/ICDManager.cpp

```diff
@@ -33,6 +33,7 @@
 #define ICD_ENFORCE_SIT_SLOW_POLL_LIMIT 0
 #endif
 
+// This can be enable by adding report_on_active to your build arguments
 #ifndef ICD_REPORT_ON_ENTER_ACTIVE_MODE
 // Enabling this makes the device emit subscription reports when transitioning from idle to active mode.
 #define ICD_REPORT_ON_ENTER_ACTIVE_MODE 0
```

Comment:
This doesn't make sense: this define is not used anywhere in this file.... why are we defining it at all?